electrumx/server/env.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, coin=None):
         self.reorg_limit = self.integer('REORG_LIMIT', self.coin.REORG_LIMIT)
         # Server stuff
         self.tcp_port = self.integer('TCP_PORT', None)
+        self.websocket_port = self.integer('WEBSOCKET_PORT', None)
```

Comment:
It's not clear if this is SSL or plain text communication.

Can we support both?  Perhaps with env vars WS_PORT and WSS_PORT?

Reply:
@kyuupichan I think the best option is check for SSL_CERTFILE and SSL_KEY, if those are specified, ElectrumX will serve a Secure WebSocket

Reply:
Updated